SConstruct

```diff
@@ -161,6 +168,10 @@ def set_defaults(env, daos_version):
         #could refine this but for now, just assume these warnings are ok
         env.AppendIfSupported(CCFLAGS=PP_ONLY_FLAGS)
 
+    if GetOption("daos_ssd"):
+		if not os.path.exists('src/control/cmd/daos_ssd'):
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 179)